src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -19,11 +19,12 @@
  * Defines all elements shown in the request section.
  */
 public class RsccRequestView extends BorderPane {
-  private final Rscc model;
-  private final Strings strings = new Strings();
+  private static final double STANDARD_SIZE = 50d;
```

Comment:
this one isn't very descriptive as well, maybe "BUTTON_SIZE" would make more sense?